qdrant-landing/content/documentation/fastembed/fastembed-quickstart.md

```diff
@@ -0,0 +1,87 @@
+---
+title: "Quickstart" 
+weight: 2
+---
+
+# How to Generate Text Embedings with FastEmbed
+
+## Install FastEmbed
+```python
+pip install fastembed
+```
+Just for demo purposes, you will use Lists and NumPy to work with sample data.
+```python
+from typing import List
+import numpy as np
+```
+
+## Load default model
+
+In this example, you will use the default text embedding model, `BAAI/bge-small-en-v1.5`. 
+```python
+from fastembed import TextEmbedding
+```
+
+## Add sample data
+
+Now, add two sample documents. Your documents must be in a list, and each document must be a string
+```python
+documents: List[str] = [
+    "FastEmbed is lighter than Transformers & Sentence-Transformers.",
+    "FastEmbed is supported by and maintained by Qdrant.",
+]
+```
+Download and initialize the model. Print a message to verify the process.
+
+```python
+embedding_model = TextEmbedding()
+print("The model BAAI/bge-small-en-v1.5 is ready to use.")
+```
+## Embed data
+
+Generate embeddings for both documents. 
+```python
+embeddings_generator = embedding_model.embed(documents)
+embeddings_list = list(embeddings_generator)
+len(embeddings_list[0])  
+```
+Here is the sample document list. The default model creates vectors with 384 dimensions.
+
+```bash
+Document: This is built to be faster and lighter than other embedding libraries e.g. Transformers, Sentence-Transformers, etc.
+Vector of type: <class 'numpy.ndarray'> with shape: (384,)
+Document: fastembed is supported by and maintained by Qdrant.
+Vector of type: <class 'numpy.ndarray'> with shape: (384,)
```

Comment:
I think we can provide just one example to save some space
By the way, the code above does not produce this output